code/Data exploration.R

```diff
@@ -90,6 +95,21 @@ data %>% filter(is.na(TOTALDEMAND)) #should return 0
 ##Temperature##
 ###############
 
+#Temperature has some questionable measures
+summary(temperature_nsw) #comes from base data
+
+#strange values start from -1643
+table(temperature_nsw %>% 
+        filter(TEMPERATURE <= 0) %>% 
+        select(TEMPERATURE) %>% 
+        arrange(TEMPERATURE))
+
+summary(temperature_nsw$TEMPERATURE)
+
+#Remove temperatures below -23 (doesn't make any sense for NSW otherwise)
+#Reference: http://www.bom.gov.au/climate/extreme/records.shtml
+
+temperature_nsw <- temperature_nsw %>% filter(TEMPERATURE >=-23)
```

Comment:
instead of removing, could we interpolate the values instead? (average of either side)

Reply:
I removed and replaced the gaps with interpolation 👍